src/js/wasm.js-post.js

```diff
@@ -177,7 +177,7 @@ function integrateWasmJS(Module) {
     info['env'] = env;
     var instance;
     try {
-      instance = Wasm['instantiateModule'](getBinary(), info);
+      instance = new WebAssembly.Instance(new WebAssembly.Module(getBinary()), info)
```

Comment:
IIUC the angry bots JS file is trying to also handle versions of wasm older than 0xc, and that the Wasm object is there to support that for now but to go away. Since Binaryen only emits 0xc now anyway, the JS should probably only support the new API.